pkg/sqlite/graphloader.go

```diff
@@ -77,6 +103,12 @@ func graphFromEntries(channelEntries []registry.ChannelEntryAnnotated) (map[stri
 			Version:    entry.Version,
 			CsvName:    entry.BundleName,
 		}
+
+		// skip synthetic channelentries that aren't pointers to actual bundles
+		if _, ok := existingBundles[key]; !ok {
```

Comment:
why are these skipped?

Reply:
This is specifically for `skips`. If the bundle is referenced but does not actually exist in the database on its own we shouldn't have a graph entry for it, just a reference in the replaces map.